### PR TITLE
Cura 7099 sync sideloaded plugins

### PR DESCRIPTION
**Pull request by ninovanhooff**
Has a Companion UM PR

### CODE REVIEW

**Comment by ninovanhooff** [comment]
> This is a singleton, so don't cache it.

@Ghostkeeper What would be the alternative approach? Do you prefer that I cache CuraApplication instead? It doesn't use the Singleton pattern, but can be considered as such.

**Comment by ninovanhooff** [comment]
> Should we turn this into a protected function then?

Thought about this, but probably not. The scenario where the user copies a plugin directly to the corresponding settings folder probably needs to use this

**Comment by ninovanhooff** [comment]
> The original line here explains why this plug-in can't be disabled: If you disable the Toolbox, you've disabled the interface to disable/enable plug-ins, so you couldn't change it back. To prevent this we've made the Toolbox a required plug-in.

Reverted

File: plugins/Toolbox/src/Toolbox.py [hunk 1]
```diff
@@ -377,7 +371,6 @@ def resetMaterialsQualitiesAndUninstall(self) -> None:
     def onLicenseAccepted(self):
         self.closeLicenseDialog.emit()
         package_id = self.install(self.getLicenseDialogPluginFileLocation())
-        self.subscribe(package_id)
```
Comment (Ghostkeeper) [inline plugins/Toolbox/src/Toolbox.py]: Hmm, I'd still expect it to subscribe also if you install it via the Marketplace? I'll test if this is still working. Seems that you intended to remove this, so I don't understand. Was this removed just for testing maybe?

**Pull request merged**